ament_cmake_cpplint/cmake/ament_cpplint.cmake

```diff
@@ -51,7 +51,7 @@ function(ament_cpplint)
   endif()
   if(ARG_FILTERS)
     string(REPLACE ";" "," filters "${ARG_FILTERS}")
-    list(APPEND cmd "--filters=${filters}")
+    list(APPEND cmd "--filter=${filters}")
```

Comment:
This is a bit out of scope, but the flag name was incorrect:

https://github.com/ament/ament_lint/blob/2d4f46c24d8d077670a3d755ee289309d3dd9c28/ament_cpplint/ament_cpplint/cpplint.py#L6797

Happy to remove an put in a separate PR, if needed.

Reply:
I think `--filters` is the correct flag because this is calling the wrapper `ament_cpplint` instead of `cpplint`

https://github.com/ament/ament_lint/blob/1f7e2b44b17bd35702143f70c4c098721912f2ee/ament_cpplint/ament_cpplint/main.py#L73-L74